queue_services/entity-filer/src/entity_filer/filing_processors/incorporation_filing.py

```diff
@@ -114,9 +125,69 @@ def _update_cooperative(incorp_filing: Dict, business: Business, filing: Filing)
         document.business_id = business.id
         document.filing_id = filing.id
         business.documents.append(document)
+
+        # create certified copy for rules document
+        rules_file = MinioService.get_file(cooperative_obj.get('rulesFileKey'))
+        rules_file_name = cooperative_obj.get('rulesFileName')
+        certified_rules_file_name = CERTIFIED_COPY_PREFIX + rules_file_name
+        certified_rules_file_key = _create_certified_copy(rules_file.data,
+                                                         business,
+                                                         certified_rules_file_name)
+
+        document = Document()
+        document.type = DocumentType.CERTIFIED_COOP_RULES.value
+        document.file_key = certified_rules_file_key
+        document.file_name = certified_rules_file_name
+        document.content_type = document.file_name.split('.')[-1]
+        document.business_id = business.id
+        document.filing_id = filing.id
+        business.documents.append(document)
+
+        # create certified copy for memorandum document
+        memorandum_file = MinioService.get_file(cooperative_obj.get('memorandumFileKey'))
+        memorandum_file_name = cooperative_obj.get('memorandumFileName')
+        certified_memorandum_file_name = CERTIFIED_COPY_PREFIX + memorandum_file_name
+        certified_memorandum_file_key = _create_certified_copy(memorandum_file.data,
+                                                              business,
+                                                              certified_memorandum_file_name)
+
+        document = Document()
+        document.type = DocumentType.CERTIFIED_COOP_MEMORANDUM.value
+        document.file_key = certified_memorandum_file_key
+        document.file_name = certified_memorandum_file_name
+        document.content_type = document.file_name.split('.')[-1]
+        document.business_id = business.id
+        document.filing_id = filing.id
+        business.documents.append(document)
```

Comment:
If I remember right we agreed to replace existing file. But we created new document type called Certified

Reply:
Yes, I saw "Return 2 documents to the UI" in the ticket and I thought we would keep both original and certified copy as it doesn't mention deletion of the original but I'd better confirm it then. Thanks.